Package.swift

```diff
@@ -88,18 +93,27 @@ let package = Package(
                 "FoundationEssentials",
                 "FoundationInternationalization",
             ],
-            path: "Sources/Foundation"),
+            path: "Sources/Foundation",
+            cSettings: wasiLibcCSettings),
 
         // _FoundationCShims (Internal)
-        .target(name: "_FoundationCShims",
-                cSettings: [.define("_CRT_SECURE_NO_WARNINGS",
-                                    .when(platforms: [.windows]))]),
+        .target(
+            name: "_FoundationCShims",
+            cSettings: [
+                .define("_CRT_SECURE_NO_WARNINGS", .when(platforms: [.windows]))
+            ] + wasiLibcCSettings
+        ),
 
         // TestSupport (Internal)
-        .target(name: "TestSupport", dependencies: [
-            "FoundationEssentials",
-            "FoundationInternationalization",
-        ], swiftSettings: availabilityMacros + concurrencyChecking),
+        .target(
+          name: "TestSupport",
+            dependencies: [
```

Comment:
```suggestion
          name: "TestSupport",
          dependencies: [
```

nit: is the indentation off here?